shipments_tracker/bin/dev

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env sh
+
+if ! gem list foreman -i --silent; then
+  echo "Installing foreman..."
+  gem install foreman
+fi
+
+exec foreman start -f Procfile.dev "$@"
```

Comment:
FIY: in my current project we have this configuration and the performance running it with `bin/dev` is excellent, this is only a suggestion, if it is useful for you apply it, if not forget about it
```
#!/usr/bin/env sh
OVERMIND_SKIP_ENV=1
export OVERMIND_SKIP_ENV

PORT="${PORT:-3000}"
export PORT

if command -v overmind > /dev/null 2>&1; then
  exec overmind start -f Procfile.dev "$@"
else
  echo "Installing foreman..."
  gem install foreman
  exec foreman start -f Procfile.dev "$@"
fi
```